modules/apps/foundation/portal-search/portal-search-web/src/main/resources/META-INF/resources/components/results/map/view.jsp

```diff
@@ -74,11 +74,17 @@ PortletURL portletURL = renderResponse.createRenderURL();
 	About <%= searchResultsAmount %> results for <strong><%= searchQuery %></strong>
 </p>
 
-<aui:row cssClass="search-map-list-container sidebar-open" id="searchMapListContainer">
-	<div class="search-map-container">
+<div class="open search-sidenav-container sidenav-container" id="<portlet:namespace />searchSideNavContainer">
+	<div class="sidenav-menu-slider">
+		<div class="sidenav-menu">
+			<%@ include file="/components/results/list/results_list.jspf" %>
```

Comment:
Nevermind, realized the file was already there since before the pull.

Trying to figure out a weird `ERROR [http-nio-8080-exec-3][MVCPortlet:551] null is not a valid include`